vm/src/obj/objbytes.rs

```diff
@@ -509,8 +509,8 @@ impl PyBytesRef {
                                     }
                                 }
                                 None => {
-                                    if replacing_char.is_some() {
-                                        decode_content.push(replacing_char.unwrap())
+                                    if let Some(replacing_char) = replacing_char {
+                                        decode_content.push(replacing_char)
```

Comment:
While the existing pattern is a little unidiomatic, I'm not sure I like how this is now the only place in the function where ```replacing_char``` is a different type.  I wanted to point it out in case you also think it might not be worthwhile.

Reply:
This looks fine to me, but you could rename the `char` version of the variable something else if you feel that it's inconsistent.